src/main/java/org/nanonative/MongoDbService.java

```diff
@@ -5,11 +5,40 @@
 import org.nanonative.nano.core.model.Service;
 import org.nanonative.nano.helper.event.model.Event;
 
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoDatabase;
+import org.nanonative.nano.services.logging.LogService;
+
+import static org.nanonative.nano.helper.config.ConfigRegister.registerConfig;
+
+
 /**
  * SEE ALSO {@link org.nanonative.nano.services.logging.LogService} for example implementation
  */
 public class MongoDbService extends Service {
 
+    public static final String CONFIG_MONGO_URI = registerConfig(
+        "mongo.uri",
```

Comment:
The naming pattern is currently `app_service_<servicename>_<key>` so it should be `app_service_mongodb_uri`